src/main/kotlin/io/github/corbym/dokker/DokkerContainer.kt

```diff
@@ -9,33 +9,36 @@ class DokkerContainer(
     var onStart: (dokker: DokkerContainer, runResponse: String) -> Unit = { _, _ -> },
 ) : DokkerLifecycle, DokkerProperties by dokkerRunCommandBuilder {
     private val withPublishedPorts = if (publishedPorts.isNotEmpty()) " with published ports $publishedPorts" else ""
+
     override fun start() {
         if (!hasStarted()) {
             debug("starting container: $name with exposed ports $expose$withPublishedPorts ")
             checkContainerStopped()
+            // force networks to exist
+            dokkerRunCommandBuilder.networks.forEach { DokkerNetwork(dokkerRunCommandBuilder.process, it).start() }
```

Comment:
are we sure we want to do that? mb its better to throw error if network is not created, it would require user to be more explicit instead of relying on hidden behaviour?

Reply:
Didn't realize I included this. :)
It came about that I got the exception thrown (as suggested) when I first started using it on a new machine and found it to be not so nice from a UX view. 
I'm looking at this as we're in the process of provisioning an environment and this line is adding a dependency if missing, so not sure if it would be hidden. Also note that `dokkerRunCommandBuilder.networks` property is user specified, so there is already intention there to add a dependency, imo.
Perhaps the comment can be revised to read: `create user requested network if missing` instead?

Reply:
Yeah I'm not sure about that either, lets not do that @sedexken - I'd rather leave the error.

Reply:
Sure, I suppose that's not the main goal of the PR anyway.